armi/reactor/blocks.py

```diff
@@ -2002,10 +1985,14 @@ def autoCreateSpatialGrids(self):
                 )
             )
 
-        spatialLocators = grids.MultiIndexLocation(grid=self.spatialGrid)
         ringNumber = hexagon.numRingsToHoldNumCells(self.getNumPins())
         # For the below to work, there must not be multiple wire or multiple clad types.
-        grid = grids.HexGrid.fromPitch(self.getPinPitch(cold=True), numRings=0)
+        # note that it's the pointed end of the cell hexes that are up (but the
+        # macro shape of the pins forms a hex with a flat top fitting in the assembly)
+        grid = grids.HexGrid.fromPitch(
+            self.getPinPitch(cold=True), numRings=0, pointedEndUp=True
```

Comment:
Is this still working if a user puts into their layout blueprint the `geom: hex_corners_up` option?

I can't quite tell myself, but it feels like maybe in that case you should set `pointedEndUp=False`?

Reply:
it only works on flat side up cases, you're right. I added a note in the docstring to the effect. It's be nice to detect that situation and throw an error, but we don't currently have a good way to look at a grid and determine which 'orientation' it is at this level.

Reply:
> we don't currently have a good way to look at a grid and determine which 'orientation'

Sad trombone noises.

Reply:
Found this researching #1284